Patterns/English/English-DateTime.yaml

```diff
@@ -986,7 +988,7 @@ HolidayNames: !dictionary
     inaugurationday: [ inaugurationday ]
     groundhougday: [ groundhougday ]
     valentinesday: [ valentinesday ]
-    stpatrickday: [ stpatrickday ]
+    stpatrickday: [ stpatrickday, stpatricksday, st.patrickday, st.patricksday ]

```

Comment:
We should add a normalization in the parser to avoid having to add all forms here.
Do you think you can do that in a future PR?

Reply:
We can discuss possible normalization logic. Well for this specific one, I'm creating a new PR to address the improvement.